src/main/kotlin/ink/abb/pogo/scraper/Bot.kt

```diff
@@ -71,6 +72,7 @@ class Bot(val api: PokemonGo, val settings: Settings) {
                 if (!settings.walkOnly) {
                     task(catch)
                     task(drop)
+                    task(evolve)
```

Comment:
Here should be: if autoEvolve is not empty run task(evolve), otherwise it will run an empty thread.